src/cff.h

```diff
@@ -25,7 +25,7 @@ struct CFFIndex {
 class OpenTypeCFF : public Table {
  public:
   explicit OpenTypeCFF(Font *font, uint32_t tag)
-      : Table(font, tag),
+      : Table(font, tag, OTS_TAG_CFF),
```

Comment:
Yes, that would work equally well. I was thinking that in principle, at least, we could have a Table subclass that handles more than one tag, in which case it would have a single canonical "Type" but potentially various tag values. So the tag and the type are (in theory) distinct; tag could be whatever the caller chooses to pass, while type is constant for a given subclass, so it made sense to me that it should be hard-coded here.

But in practice I don't think that happens: with things like hmtx/vmtx, which share their format, they're handled as distinct (but trivial) subclasses of OpenTypeMetricsTable.